hoi_minim/cogs/demons.py

```diff
@@ -308,6 +308,18 @@ async def _vtuber_copypasta(
 
             await ctx.reply(file=discord_file, mention_author=False)
 
+    @commands.command("max_pity")
+    async def max_pity(
+        self, ctx: Context, max_pity: Annotated[int, commands.Range[0, 100]]
+    ):
+        if ctx.author.id == self.bot.owner_id:
```

Comment:
there's a decorator to restrict commands to owner-only

```suggestion
    @commands.command("max_pity")
    @commands.is_owner()
    async def max_pity(
        self, ctx: Context, max_pity: Annotated[int, commands.Range[0, 100]]
    ):
```